ktls/src/stream.rs

```diff
@@ -0,0 +1,251 @@
+//! See [`KtlsStream`].
+
+#![allow(clippy::module_name_repetitions)]
+
+pub mod context;
+pub mod error;
+pub mod impl_std;
+#[cfg(feature = "async-io-tokio")]
+pub mod impl_tokio;
```

Comment:
Neither of these modules actually contain any public types.

Reply:
I moved them to separate modules just to make `stream.rs` looks cleaner 🤔

Reply:
The issue isn't that they are in separate modules (that's fine). It's that they are public but have no public types. They should just be private.